comments/classified.csv

```diff
@@ -2626,3 +2626,199 @@ Note: this affects *public* APIs we've shipped.  These APIs would now seemingly
 @github Not sure how to tell the difference. Probably both,0,0.5
 "Assuming that analyzer package is referenced by compiler toolset (correct me if that's wrong), there is no compiler toolset which references the new analyzer package that I need (since it's brand new)",0,0.5
 
+Is this actually needed,0,0.5
+"In base64 the encoded data is stored as big endian like, and here it's written by https://github.com/dotnet/runtime/blob/44595c05ca033d1010397d05a55ab89bf078e56c/src/libraries/System.Memory/src/System/Buffers/Text/Base64Encoder.cs#L95 (Edit: this is wrong ~#code) and not as #code). So I believe this change flips the order.~",0,0.5
```

Comment:
These two lines are in this file a couple times. Did something happen and we didn't remove lines from `potential-bad-02.csv`?

Reply:
let me check, whats going on